application-templates/typescript/assets/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "assets-typescript",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite --port 3000",
```

Comment:
Let's keep the Vite default port, there is no apparent reason to change it to 3000.